README.md

```diff
@@ -6,6 +6,7 @@
 [![PyPI - Status](https://img.shields.io/pypi/status/mbed-tools-ci-scripts)](https://pypi.org/project/mbed-tools-ci-scripts/)
 [![PyPI - Python Version](https://img.shields.io/pypi/pyversions/mbed-tools-ci-scripts)](https://pypi.org/project/mbed-tools-ci-scripts/)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/ARMmbed/mbed-tools-ci-scripts/blob/master/LICENSE)
```

Comment:
I think is may be better to add a new line here so license information is on it's own line.

Reply:
I made the change but I don't know if it was exactly what you wanted.

Reply:
I meant before license badge so they were both on the same line.

Reply:
ok